galaxy.yml

```diff
@@ -0,0 +1,107 @@
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: cifmw
```

Comment:
beware: we may need to update all the roles "meta/main.yml" to reflect that namespace. And the skeleton for new roles.

Reply:
right, I see that roles use the `edpm`  namespace, should I changed it here to `edpm` or change the roles one to `cifmw`?

Reply:
cifmw makes more sense. I chose the "edpm" more because... "why not converging in the same namespace", but we may want our own.

Reply:
ack, I renamed all the roles namespaces to `cifmw` in a second commit, so it can be easily reverted